desc/equilibrium/equilibrium.py

```diff
@@ -858,6 +858,42 @@ def compute(  # noqa: C901
             msg="must pass in a Grid object for argument grid!"
             f" instead got type {type(grid)}",
         )
+        # a check for Redl to prevent computing on-axis or
+        # at a rho=1.0 point where profiles vanish
+        if any(["Redl" in name for name in names]):
```

Comment:
I don't think this check is necessary or desired. We have plenty of other compute quantities that are sometimes undefined and we don't bother to check for them all here. `eq.compute` is mostly used for post-processing so a few NaN values for plotting etc. isn't problematic like it is in the optimization. We should let users compute the quantities they want and handle NaN values on their own, since that could be useful in some cases.

Reply:
my rationale is that Redl is often used for the iterative method which bypasses all the checks we put in the `BootstrapRedlConsistency` objective, so I wanted to also have the checks in the compute method. 

I alternatively could make it a warning?

Reply:
Definitely don't throw an error, because otherwise you can't even call the compute method with that grid. 

I just feel like it is adding a lot of code for a really marginal use case. In practice I think the only reason to use the iterative method over an optimization is if you have a spline profile. And if people are copying the example from the tutorial, they should also be reading the explanation about what grid to use. 

If you really want to throw a warning in `eq.compute`, I would prefer something more general that covers all quantities. Like we could check if any values in `data` are NaN, and throw a warning saying something like "Some of the computed quantities are not defined, check the Grid you are computing on."

Reply:
I'll do a warning but still just for Redl. I think using a generic NaN catch will cause a lot of areas of the code to start throwing the warning which I don't want to deal with in this PR. Making an issue for it